src/init.cpp

```diff
@@ -943,6 +948,15 @@ bool AppInit2(ThreadHandlerPtr threads)
         NN::Quorum::LoadSuperblockIndex(pindexBest);
     }
 
+    // Initialize the Gridcoin research reward tally system from the first
+    // research age block (as defined in main.h):
+    //
+    uiInterface.InitMessage(_("Initializing research reward tally..."));
+    if (!NN::Tally::Initialize(BlockFinder().FindByHeight(fTestNet ? 36501 : 364501)))
```

Comment:
These are also used in `IsResearchAgeEnabled`. Perhaps it could be a good idea to extract them to a `GetResearchAgeHeight()` or something.

Reply:
Ah, yeah, good idea.